src/backend/src/controllers/teams.controllers.ts

```diff
@@ -62,4 +62,16 @@ export default class TeamsController {
       next(error);
     }
   }
+
+  static async setTeamLead(req: Request, res: Response, next: NextFunction) {
+    try {
+      const { userId } = req.body;
```

Comment:
Nice work overall on setting the controller.

We want to update this to be userIds (array of user ids) after the edit in the router file.